src/com/intellij/idea/plugin/hybris/impex/formatting/ColumnsAlignmentStrategy.kt

```diff
@@ -0,0 +1,91 @@
+/*
+ * This file is part of "hybris integration" plugin for Intellij IDEA.
+ * Copyright (C) 2014-2016 Alexander Bartash <AlexanderBartash@gmail.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package com.intellij.idea.plugin.hybris.impex.formatting;
+
+import com.intellij.formatting.Alignment
+import com.intellij.idea.plugin.hybris.impex.psi.ImpexFile
+import com.intellij.idea.plugin.hybris.impex.psi.ImpexTypes
+import com.intellij.idea.plugin.hybris.impex.psi.ImpexUserRightsValueGroup
+import com.intellij.idea.plugin.hybris.impex.psi.ImpexValueGroup
+import com.intellij.idea.plugin.hybris.impex.utils.ImpexPsiUtils
+import com.intellij.lang.ASTNode
+import com.intellij.psi.util.PsiTreeUtil
+import org.jetbrains.annotations.Contract
+
+open class ColumnsAlignmentStrategy : AlignmentStrategy {
+
+    private val alignments: MutableList<Alignment> = ArrayList();
+    private var columnNumber = 0;
+
+    override fun getAlignment(currentNode: ASTNode): Alignment {
+        if (!isNewColumn(currentNode)) return Alignment.createAlignment()
+        if (columnNumber < alignments.size) return alignments[columnNumber]
```

Comment:
logic is different. previously `columnNumber` was always increased if not a new column and after all logic
<img width="403" alt="image" src="https://github.com/absinthminded/sap-commerce-intellij-idea-plugin/assets/2292510/86936b61-134d-4305-be6d-2492a057fd85">